app/models/user.rb

```diff
@@ -1,11 +1,16 @@
 class User < ApplicationRecord
+  # Include default devise modules. Others available are:
+  # :confirmable, :lockable, :timeoutable, :trackable and :omniauthable
+  devise :database_authenticatable, :registerable,
+         :recoverable, :rememberable, :validatable
```

Comment:
- [ ] Please make sure that `:confirmable` is enabled in the user model :point_down: 
![Screenshot from 2022-06-24 16-11-35](https://user-images.githubusercontent.com/67087664/175523563-3422717b-be5f-43ab-b1b5-d76fea3eb142.png)

Reply:
@elyor-doniyorov Thanks for reminding me. I was facing a lot of trouble because I miss it but didn't sure it was a case. big respect<img src="https://media1.giphy.com/media/RRw0Z2GL8hKC5uj0NL/giphy.gif"/>